common/translations/lang/en.json

```diff
@@ -1,6 +1,15 @@
 {
   "code": "en",
   "data": {
+    "ADDRESS": "Address",
+    "LABEL": "Label",
+    "INVALID_LABEL_LENGTH": "An address label must be between 2 and 20 characters.",
```

Comment:
Why are we imposing a character limit for labels? If responsiveness is an issue, we can always truncate the labels.

Reply:
Also, addresses should be truncated on smaller screens. Personally I like the way metamask does it, they truncate the middle section of an address so you can verify the first 3 and last 3 digits of an address.

Reply:
I imposed the limit because I was asked to by will. Truncation is fine, but again, in the interest of avoiding story bloat that more advanced functionality might be better as a different story.

Reply:
Fixed: Updated to 50 and introduced truncation in the various spots that display the label.